app/src/main/java/ch/epfl/sdp/contamination/Layman.java

```diff
@@ -24,6 +25,10 @@ public Layman(InfectionStatus initialStatus) {
     public Layman(InfectionStatus initialStatus, float infectedWithProbability) {
         this(initialStatus, infectedWithProbability, "__NOT_UNIQUE_NOW");
     }
+    @VisibleForTesting
```

Comment:
Why only this one is visibleForTesting?

Reply:
Because I saw you use them, and I find it cool. Ok I remove it.